lib/ansible/modules/iptables.py

```diff
@@ -531,6 +531,14 @@
     log_prefix: "IPTABLES:INFO: "
     log_level: info
 
+- name: Block outgoing traffic to a specific IP
+  ansible.builtin.iptables:
+    chain: OUTPUT
```

Comment:
There is already an example of `INPUT` above. This adds almost no value to the documentation.